docs/api.md

```diff
@@ -81,6 +81,7 @@
   * [page.browser()](#pagebrowser)
   * [page.click(selector[, options])](#pageclickselector-options)
   * [page.close(options)](#pagecloseoptions)
+  * [page.closed](#pageclosed)
```

Comment:
let's do a `page.isClosed()` method instead of the getter to follow  [API Guidelines](https://github.com/GoogleChrome/puppeteer/blob/master/CONTRIBUTING.md#api-guidelines).